src/controllers/v1/comments.js

```diff
@@ -42,12 +43,21 @@ module.exports = class comments {
 	 */
 	async update(req) {
 		try {
-			const comment = await commentService.update(
-				req.params.id ? req.params.id : '',
-				req.query.resource_id,
-				req.body,
-				req.decodedToken.id
-			)
+			let comment
+			if (req.method === common.REQUEST_METHOD_DELETE) {
+				comment = await commentService.delete(
+					parseInt(req.params.id),
```

Comment:
@adithyadinesh0412 do we need to convert to int? We can validate in the validator and throw error from there if its not a number

Reply:
But the values are coming as string in the service and db is not able to find the right data. So I'm type casting it. @priyanka-TL